MiniTwitch.Helix/Requests/ChannelStreamScheduleSegmentBody.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MiniTwitch.Helix.Requests;
+
+public readonly struct ChannelStreamScheduleSegmentBody
```

Comment:
why `struct`? (applies to all API models)

Reply:
I think it's slightly helpful for allocations? It makes creating request objects that are meant for later use less expensive, I think that's reasonable?